SpatialGDK/Source/SpatialGDKEditorToolbar/Private/SpatialGDKEditorToolbar.cpp

```diff
@@ -872,14 +903,7 @@ bool FSpatialGDKEditorToolbarModule::StartLocalSpatialDeploymentIsVisible() cons
 
 bool FSpatialGDKEditorToolbarModule::StartLocalSpatialDeploymentCanExecute() const
 {
-	if (LocalDeploymentManager->IsSpatialServiceRunning())
-	{
-		return !LocalDeploymentManager->IsLocalDeploymentRunning();
-	}
-	else
-	{
-		return true;
-	}
+	return !LocalDeploymentManager->IsServiceStarting() && !LocalDeploymentManager->IsDeploymentStarting();
```

Comment:
this has already been fixed in a different PR, right?

Reply:
Already been fixed in this PR: 
`https://github.com/spatialos/UnrealGDK/pull/2112`

Reply:
How about leave it here and process it when merging back.

Reply:
ah no, that's ok. It's already fixed in the other pr, so no need to do it again :)